### PR TITLE
QA: Overlay 

### PR DESCRIPTION
**Pull request by chrismekim**
#53 

Preview:
Screen Shot 2020-04-07 at 2 26 28 PM

### CODE REVIEW

File: src/styles/_global.scss [hunk 1]
```diff
@@ -10,3 +10,14 @@
     grid-template-columns: repeat(8, 1fr);
   }
 }
+
+.overlay-is-active {
+  height: 100%;
```
Comment (chrismekim) [inline src/styles/_global.scss:15]: Created these classes to add to html's classList if the overlay is active or not.

File: public/index.html [hunk 2]
```diff
@@ -35,6 +35,27 @@
 
     gtag('config', 'UA-88778470-4');
   </script>
+  <script>
```
Comment (chrismekim) [inline public/index.html:38]: Working but not sure if this is the best way to do this!

File: public/index.html [hunk 3]
```diff
@@ -35,6 +35,27 @@
...
+        if (html && html.classList.contains('overlay-is-active')) {
+          html.classList.remove('overlay-is-active');
+          html.classList.add('overlay-is-inactive')
+        }
+
+      } catch (e) {
+        return;
+      }
+    }
+  </script>
```
Comment (hhff) [inline public/index.html:58]: This function isn't being called anywhere in this PR?

File: src/styles/_global.scss [hunk 4]
```diff
@@ -10,3 +10,14 @@
...
+  background-color: color('gray-darkest');
+  overflow: hidden;
+}
+
+.overlay-is-inactive {
+  background-color: color('white');
+  overflow: auto;
+}
```
Comment (hhff) [inline src/styles/_global.scss:23]: This should be the default state of the document. It shouldn't be a class. `overlay-is-active` should just override it!

Also, the above function isn't being called anywhere, so this class is never used in this code

File: src/views/MainView.js [hunk 5]
```diff
@@ -36,15 +36,20 @@ class MainView extends PureComponent {
...
     const now = new Date();
     const currentTimeInHours = now.getHours();
     
     const shouldShowOverlay =
       currentTimeInHours < dayStartTimeInHours || currentTimeInHours >= dayEndTimeInHours;
 
-      return shouldShowOverlay;
+    if (shouldShowOverlay && html && !html.classList.contains('overlay-is-active')) {
+      html.classList.add('overlay-is-active');
+    };
```
Comment (hhff) [inline src/views/MainView.js:48]: this should remove `overlay-is-active` when shouldShowOverlay is false.

It should be more like:

```
if (shouldShowOverlay) {
  // ensure `overlay-is-active` is on the document
} else {
  // ensure `overlay-is-active` is removed from the document
}
```

File: public/index.html (hunk 3 above)
Comment (hhff) [reply public/index.html:58]: Let's remove it - it's not necessary

**Pull request merged**